Source/PartnerCenter.CustomerPortal/BusinessLogic/Commerce/PaymentGateways/PayUMoney/ApiCalls.cs

```diff
@@ -0,0 +1,100 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="ApiCalls.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace PayUMoney.Api
+{
+    using System;
+    using System.Collections.Specialized;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
+    using Microsoft.Store.PartnerCenter.CustomerPortal.BusinessLogic;
+    using Microsoft.Store.PartnerCenter.CustomerPortal.Models;
+
+    /// <summary>
+    /// class definition
+    /// </summary>
+    public class ApiCalls
```

Comment:
>ApiCalls [](start = 17, length = 8)

I agree with @isaiah.williams@microsoft.com 

We should be initializing a new instance of the HttpClient every time we perform an operation. I would recommend that we create a static instance of it either for this class or globally in the Application Domain. Check out 
[https://docs.microsoft.com/en-us/azure/architecture/antipatterns/improper-instantiation/](https://docs.microsoft.com/en-us/azure/architecture/antipatterns/improper-instantiation/) for more details as to why I am making this recommendation.

Reply:
Done.

---
In reply to: [183244460](https://github.com/PartnerCenterSamples/Reseller-Web-Application/pull/61#discussion_r183244460) [](ancestors = 183244460)